discovery_engine_core/providers/src/client.rs

```diff
@@ -35,24 +35,28 @@ pub enum Error {
 }
 
 /// Client that can provide documents.
+#[derive(Default)]
 pub struct Client {
     token: String,
     url: String,
 }
 
 /// Parameters determining which news to fetch
 pub struct NewsQuery {
-    market: Market,
-    filter: Filter,
+    /// Market of news.
+    pub market: Market,
+    /// News filter.
+    pub filter: Filter,
     /// How many articles to return (per page).
-    page_size: Option<usize>,
+    pub page_size: Option<usize>,
```

Comment:
Is that really optional for us?

(same for the other query)

Reply:
It was a some point. Probably now not anymore.

Reply:
But we can change it later.